modules/@apostrophecms/login/ui/apos/components/TheAposLogin.vue

```diff
@@ -245,12 +275,14 @@ export default {
     &__footer {
       z-index: $z-index-manager-display;
       position: fixed;
+      right: 0;
       bottom: 32px;
-      left: 32px;
+      left: 0;
       display: flex;
+      width: fit-content;
```

Comment:
In FF, this rule is invalid, causing the logo and the version to be stuck on the right and on the left of the screen without padding/margin. I would change `position: fixed` to `position: absolute` and remove `right` and `left`rule. 

Also, the version is the project's version not Apostrophe's. If the original idea was to put forward the version of Apostrophe used in the project, I have to make a code change. If it was for the version of the project (label "version" in package.json), I suggest to move the version under or next to the project's name (in `apos-login__header`)  and to leave in the footer only the logo and the label "Apostrophe CMS", both centered in the same width as the body (320px).

Reply:
Yes, my thought was for it to be the current version of Apostrophe, which feels like it makes sense with the CMS brand footer.